lib/cypress_on_rails/vcr_wrapper.rb

```diff
@@ -0,0 +1,38 @@
+require 'rack'
+require 'cypress_on_rails/configuration'
+
+module CypressOnRails
+  class VCRWrapper
+
+    def initialize(app:, env:)
+      @app = app
+      @env = env
+      @request = Rack::Request.new(env)
+    end
+
+    def run_with_cassette
+      VCR.use_cassette(cassette_name, { :record => configuration.vcr_record_mode }) do
```

Comment:
Not sure 🤔 if this will work as it only wraps one request.

I think we actually need two endpoints, one to inject a cassette and one to eject a cassette.

Reply:
@grantspeelman As the doc says https://www.rubydoc.info/gems/vcr/VCR:insert_cassette They recommend to use `use_cassette` instead of pair of insert-eject calls. 
I guess `use_cassette` block should work the same way. The only change here is wrapping "normal" app flow. 
Why do you think that insert/eject will perform differently?
I tested it on the real project under `test` env, and I've got a cassette written with 2 API calls inside.

Reply:
guess i just don't see how "use_cassette" approach will work, but maybe once you have a working example it will make sense 👍🏽

Reply:
I, it's making sense now. 
Every single request it wrapped in it's own VCR cassette 👍🏽 and it's not really controlled from within Cypress.
This a is a good start and could help some teams around certain scenarios.
Maybe all that is still needed from Cypress is the ability to set a cassette prefix so that different scenarios can use different cassettes.

Reply:
@grantspeelman I was thinking about some sort of temporary key-value storage, that users can setup in the config and populate values using `appCommand` during the test. This storage could be reseted the same way as we do it for DB before/after each test. In that perspective, we could pass the whole name for cassette, not just a prefix.
But maybe it's too much.

Reply:
the whole name for cassette sounds like a good idea, also gets around the issue of having to try and generate a cassette name. Like that idea 👍🏽